pkg/clients/zone/zone_test.go

```diff
@@ -0,0 +1,140 @@
+package zone
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws/awserr"
+	"github.com/aws/aws-sdk-go-v2/service/route53"
+)
+
+func Test_getRegion(t *testing.T) {
```

Comment:
I'd prefer to stick to camel case [per Go convention](https://golang.org/doc/effective_go.html#mixed-caps) even in test names. I'd go with `TestGetRegion` here.

Reply:
My bad here. The mentioned test has been renamed accordingly.